lib/inferno/dsl/fhir_evaluation.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '../ext/fhir_models'
+
+module Inferno
+  module DSL
+    # This module contains the methods needed to configure a validator to
```

Comment:
This comment seems to have been copy & pasted from Validator? For now I almost think it's not worth adding comments since we'll likely be refactoring once we decide how we want to integrate everything together